src/main/resources/mariadb.sql

```diff
@@ -3,6 +3,7 @@ CREATE TABLE FAILSAFE_TASK (
     PARAMETER MEDIUMTEXT,
     NAME VARCHAR(200) NOT NULL,
     PLANNED_EXECUTION_TIME TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    NODE_ID VARCHAR(48),
```

Comment:
if node id has max length of 48 then I suggest to add a length validation at the beginning of the start method. throw an exception if too long